Makefile

```diff
@@ -906,7 +923,7 @@ mobile-run: deps-common
 	echo -e "\033[92mRunning:\033[39m mobile app"
 	$(MAKE) -C mobile run
 
-mobile-build: | deps-common
+mobile-build: | deps-common status-go
```

Comment:
The build of `status-go` needs to be called from `mobile/Makefile`, since otherwise the calls to `make` inside of `mobile` directory will not work.